docs/source/getting-started/quickstart.rst

```diff
@@ -132,6 +132,25 @@ This may show multiple clusters, if you have created several:
 
 See here for a list of all possible :ref:`cluster states <sky-status>`.
 
+Access the dashboard
+====================
+
+If you install SkyPilot from the official package, after starting the API server (when running locally, the first CLI or SDK call will automatically start a SkyPilot API server), run :code:`sky dashboard` to access the dashboard. This automatically opens a browser tab to show the dashboard for the clusters and managed jobs status.
+
+If you install SkyPilot from source, before starting the API server, refer to the `Dashboard README <https://github.com/skypilot-org/skypilot/blob/master/sky/dashboard/README.md#access-the-dashboard>`_ to build the dashboard. Then you can start your API server and access the dashboard through :code:`sky dashboard`.
+
+The clusters page example:
+
+.. image:: ../images/dashboard-clusters.png
+  :width: 800
+  :alt: Clusters dashboard
+
+The managed jobs page example:
+
+.. image:: ../images/dashboard-managed-jobs.png
+  :width: 800
+  :alt: Managed jobs dashboard
```

Comment:
Let's use some cloud instead of pure Kubernetes, e.g. showing a AWS cluster with L4 GPUs, a GCP cluster with A100 GPU, and a Kubernetes using H200 (we can mock this H200 without actually creating H200 GPUs), etc. Also, the name could be more informative, e.g. clusters: `dev`, `train`; jobs: `process`.

Reply:
Updated